### PR TITLE
searchincidentsv2: limit number of events

### PR DESCRIPTION
**Pull request by dorschw**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/46740

## Description
added argument to limit the number of event items returned in incidents. default is 0 = all.

## Must have
- [x] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CommonScripts/ReleaseNotes/1_6_36.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+##### SearchIncidentsV2
+- Added the `trim_events` argument, to limit the number of events returned from incidents.
```
Comment (yaakovi) [inline Packs/CommonScripts/ReleaseNotes/1_6_36.md:4]:
```suggestion
- Added the *trim_events* argument to limit the number of events returned from incidents.
```

File: Packs/CommonScripts/Scripts/SearchIncidentsV2/SearchIncidentsV2_test.py [hunk 2]
```diff
@@ -75,6 +76,7 @@ def test_is_incident_id_valid(id_value, expected_output):
         u'id': u'1',
         u'type': u'TypeA',
         u'name': u'Phishing',
+        u'events': ['event1', 'event2', 'event3']
```
Comment (yaakovi) [inline Packs/CommonScripts/Scripts/SearchIncidentsV2/SearchIncidentsV2_test.py:79]: [PEP8](https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas)
```suggestion
        u'events': ['event1', 'event2', 'event3'],
```

File: Packs/CommonScripts/Scripts/SearchIncidentsV2/SearchIncidentsV2.yml [hunk 3]
```diff
@@ -105,6 +105,15 @@ args:
   name: page
   required: false
   secret: false
+- default: false
+  description: |-
+    The number of events to return from the alert JSON. The default is 0, which returns all events.
+    Note that the trimming is from the top of the list, regardless of event time or other properties.
```
Comment (yaakovi) [inline Packs/CommonScripts/Scripts/SearchIncidentsV2/SearchIncidentsV2.yml:111]: `top of the list` is ambiguous.
```suggestion
    Note that the trimming is from the end of the list, regardless of event time or other properties.
```

File: Packs/CommonScripts/Scripts/SearchIncidentsV2/SearchIncidentsV2_test.py (hunk 2 above)
Comment (dorschw) [reply Packs/CommonScripts/Scripts/SearchIncidentsV2/SearchIncidentsV2_test.py:79]: it isn't actually used in the test, I'll remove it instead.

**Review by yuvalbenshalom — APPROVED** [review]

**Review by BenJoParadise — APPROVED** [review]

**Review by BenJoParadise — CHANGES_REQUESTED** [review]
A few comments:
- the parameter is called `trimevents` not `trimevent` - my bad, i relayed it wrong the first time.
- the parameter should be optional, and should not be passed to getIncident if it was not entered.

**Review by BenJoParadise — APPROVED** [review]

**Pull request merged**